AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2256,41 +2319,48 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
   return [[ASLayoutSpec alloc] init];
 }
 
+- (void)setLayoutSpecBlock:(ASLayoutSpecBlock)layoutSpecBlock
+{
+  // For now there should never be a overwrite of layoutSpecThatFits: and a layoutSpecThatFitsBlock: be provided
+  ASDisplayNodeAssert(!(_methodOverrides & ASDisplayNodeMethodOverrideLayoutSpecThatFits), @"Overwriting layoutSpecThatFits: and providing a layoutSpecBlock block is currently not supported");
```

Comment:
We already track that on multiple places. I don't think we should add some kind of reference to it at the moment.